src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/endtoendtests/helper/ZipFileHelper.java

```diff
@@ -37,7 +37,7 @@ private ZipFileHelper() {
     public static ZipArchive createZipArchive(String dirName, Container container) throws Exception {
         List<String> files =
             Stream.of(new File(getResource(dirName).getPath()).listFiles())
-                .map(File::getName)
+                .map(e -> dirName + "/" + e.getName())
```

Comment:
would `File::getPath` work?

Reply:
it would be real path. we need classpath.